acceptance/tests/resource/service/windows.rb

```diff
@@ -44,12 +44,17 @@ def service_manifest(name, params)
     :stop_sleep => 40,
   }
 
+  newUser = "tempUser#{rand(999999).to_i}"
+
   teardown do
     delete_service(agent, mock_service_long_start_stop[:name])
+
+    on(agent, puppet("resource user #{newUser} ensure=absent")) do |result|
+      assert_match(/User\[#{newUser}\]\/ensure: removed/, result.stdout)
```

Comment:
we shouldn't assert the output of puppet resource here, as the test might fail before it even creates the user, causing this assertion to fail